lib/mindwendel/brainstormings/idea.ex

```diff
@@ -12,21 +13,27 @@ defmodule Mindwendel.Brainstormings.Idea do
   schema "ideas" do
     field :body, :string
     field :username, :string, default: "Anonymous"
-    field :label, Ecto.Enum, values: @label_values
+    field :label_old, Ecto.Enum, source: :label, values: @label_values
```

Comment:
do we really need to keep this?

Reply:
and maybe better wording: `deprecated_label`

Reply:
Yes, it is needed for the migration. But you are right, we need to move the migration file.

Reply:
I changed this.